airflow/models.py

```diff
@@ -3444,7 +3458,7 @@ def find(dag_id=None, run_id=None, execution_date=None,
         return dr
 
     @provide_session
-    def get_task_instances(self, state=None, session=None):
+    def get_task_instances(self, state=None, to_list=True, session=None):
```

Comment:
Nit: this is backwards incompatible, and could lead to issues if someone was using unnamed params and providing a session. Worth a quick glance, but I'm OK with this change.